### ISSUE TITLE
Transaction chunking

### ISSUE THREAD

**Issue by cjpearson** [brody2consult/cordova-sqlite-storage#534]
Transaction chunking was a feature of https://github.com/litehelpers/Cordova-sqlite-evplus-legacy-free, but that project is now considered legacy. Are there plans to add this feature to any of the active branches?

**Comment by brody2consult** [comment]
This is under future consideration. Someone reported problems with big batches. I think the hardest part will be automatic verification that it works properly.

**Comment by cjpearson** [comment]
After looking into it further, I think Multi-part transactions is the feature I'm looking for. Is that also being considered?

**Comment by brody2consult** [comment]
Yes this is under consideration for a new evplus version. It will probably be added without per-SQL statement callbacks. For priority feature requests you may contact sales@litehelpers.net for an estimate.
